seed/static/seed/js/services/analyses_service.js

```diff
@@ -22,12 +22,6 @@ angular.module('BE.seed.service.analyses', [])
         });
       };
 
-      let get_analysis_for_org = function (analysis_id, org_id) {
-        return $http.get('/api/v3/analyses/' + analysis_id + '?organization_id=' + org_id).then(function (response) {
-          return response.data;
-        });
-      };
-
```

Comment:
I think we wanted to actually keep this method:
<img width="912" alt="Screen Shot 2020-12-21 at 1 59 54 PM" src="https://user-images.githubusercontent.com/30608004/102812470-09e32000-4395-11eb-87d8-085ae3334a68.png">
<img width="911" alt="Screen Shot 2020-12-21 at 1 59 37 PM" src="https://user-images.githubusercontent.com/30608004/102812471-09e32000-4395-11eb-8d68-cc4295ae6bc6.png">

Reply:
Analysis vs Analyses has gotten me so many times during all these tickets..